components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -58,7 +58,7 @@ export default function () {
   const { teams } = useContext(TeamsContext);
   const location = useLocation();
   const team = getCurrentTeam(location, teams);
-  const routeMatch = useRouteMatch<{ teamSlug: string, projectSlug: string }>("/:teamSlug/:projectSlug/configure");
+  const routeMatch = useRouteMatch<{ teamSlug: string, projectSlug: string }>("/t/:teamSlug/:projectSlug/configure");
```

Comment:
I'm afraid there are more instances of `useRouteMatch` to adjust, e.g. in Projects.tsx

Reply:
Ok, thanks. I have updated those. Any other leftovers you see? 
It's a little hard to find all the references ...

Reply:
just searched for all `useRouteMatch` usages and it looks good now.

Reply:
I need to make the `/t` optional however. As the page needs to work with user projects as well.